data/application.css

```diff
@@ -25,38 +25,70 @@ textview.terminal selection {
     color: #252e32;
 }
 
-.ext2, .ext3, .ext4 {
-    background-color: rgb(114, 147, 203);
+.disk-bar {
+    padding: 0;
+}
+
+.ext2, .ext3, .ext4, .fat16, .fat32, .btrfs, .xfs, .ntfs, .luks, .lvm, .none, .swap {
+    border-width: 3px;
+    border-style: solid;
```

Comment:
Setting this like this would be better:

```
border-left-style: solid;
border-right-style: solid;
border-top-style: none;
border-bottom-style: none;

Reply:
How about `border-width: 0 3px`?

Reply:
That would work. Do 2px though.